src/main.rs

```diff
@@ -0,0 +1,65 @@
+use dirs;
```

Comment:
We could refactor this out a bit into separate functions by step. But for now, this just works as intended.